crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -1413,12 +1413,6 @@ where
             }
             Err(err) => {
                 warn!(target: "consensus::engine", ?err, "Failed to insert downloaded block");
```

Comment:
do we not need this anymore because of the fix below?

Reply:
this is just a precaution, 
in case there are more bugs with p2p downloading+validating, we want to prevent blocks downloaded from the network from being marked as invalid. 

This prevents false positives that would cause issues with new blocks sent by the CL

Reply:
gotcha

Reply:
we'll need to add this back eventually, I think this would cause some hive tests to fail, but for now we. can remove. Let's make an issue for adding whatever testing we would need to become more confident in adding this back

Reply:
> we'll need to add this back eventually, I think this would cause some hive tests to fail

oh wasn't aware, will add it back, should be fine now I think

Reply:
running tests now just to make sure, will send update with results